.github/workflows/pytest-gpu.yml

```diff
@@ -46,7 +46,7 @@ jobs:
         RESOURCE_GROUP_REGION: uksouth
```

Comment:
@rhodrin I just noticed this line here: So I would suspect that it could be straightforward if we also save an image in that region.

Reply:
Well the image wouldn't be in the required resource group - so that'd have to be set up first.